R/outpack_index.R

```diff
@@ -55,45 +56,59 @@ outpack_index <- R6::R6Class(
   ))
 
 
-index_update <- function(root_path, prev, skip_cache) {
+index_update <- function(root_path, prev, skip_cache, progress) {
+  progress <- progress %||% getOption("orderly_index_progress", TRUE)
```

Comment:
What does `%||%` do ? Does it override `progress` only when null? Is that a standard thing?